osidb/migrations/0174_add_history_to_more_models.py

```diff
@@ -0,0 +1,1068 @@
+                    when="BEFORE",
+                ),
+            ),
+        ),
+        pgtrigger.migrations.AddTrigger(
+            model_name="flawreferenceaudit",
+            trigger=pgtrigger.compiler.Trigger(
+                name="append_only",
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    func="RAISE EXCEPTION 'pgtrigger: Cannot update or delete rows from % table', TG_TABLE_NAME;",
+                    hash="37c5f43e6fc45b2f5a64873446ec537be28cf01c",
+                    operation="UPDATE OR DELETE",
+                    pgid="pgtrigger_append_only_0a80c",
+                    table="osidb_flawreferenceaudit",
+                    when="BEFORE",
+                ),
+            ),
+        ),
+        pgtrigger.migrations.AddTrigger(
+            model_name="flawcvssaudit",
+            trigger=pgtrigger.compiler.Trigger(
+                name="append_only",
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    func="RAISE EXCEPTION 'pgtrigger: Cannot update or delete rows from % table', TG_TABLE_NAME;",
+                    hash="344f9da9a58157f5ebad3b1102bd0e7d62be41dc",
+                    operation="UPDATE OR DELETE",
+                    pgid="pgtrigger_append_only_d6d29",
+                    table="osidb_flawcvssaudit",
+                    when="BEFORE",
+                ),
+            ),
+        ),
+        pgtrigger.migrations.AddTrigger(
+            model_name="flawcommentaudit",
+            trigger=pgtrigger.compiler.Trigger(
+                name="append_only",
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    func="RAISE EXCEPTION 'pgtrigger: Cannot update or delete rows from % table', TG_TABLE_NAME;",
+                    hash="9df59ab1989d89ffcbdfc35ae9cb4f7fd57f0f0b",
+                    operation="UPDATE OR DELETE",
+                    pgid="pgtrigger_append_only_2d4f0",
+                    table="osidb_flawcommentaudit",
+                    when="BEFORE",
+                ),
+            ),
+        ),
+        pgtrigger.migrations.AddTrigger(
+            model_name="flawacknowledgmentaudit",
+            trigger=pgtrigger.compiler.Trigger(
+                name="append_only",
+                sql=pgtrigger.compiler.UpsertTriggerSql(
+                    func="RAISE EXCEPTION 'pgtrigger: Cannot update or delete rows from % table', TG_TABLE_NAME;",
+                    hash="7aa6d5750a7547629ba6e7d619f8ce395711dc4b",
+                    operation="UPDATE OR DELETE",
+                    pgid="pgtrigger_append_only_1ca74",
+                    table="osidb_flawacknowledgmentaudit",
+                    when="BEFORE",
+                ),
+            ),
+        ),
+        # Enable row level security for the new history models
+        migrations.RunSQL(
+            reverse_sql=migrations.RunSQL.noop,
+            sql="""
+    ALTER TABLE osidb_snippetaudit FORCE ROW LEVEL SECURITY;
```

Comment:
I think we need to add this to etc/pg/setup-rls-audit-tables.sql as well ?

Reply:
I guess? I'm not sure what that file is used for. Isn't it sufficient with running the migration?

Reply:
maybe this is no longer used (I think this was originally just used by docker for dev purposes) ... might be worth checking previous migrations take care of it (and removing)